### PR TITLE
Add unit tests

### PR DESCRIPTION
**Pull request by lvdoorn**
In this branch configuration for qunit tests has been added, as well as a test suite for the pullrequest transformer.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,20 +5,24 @@
     "node": ">= 0.8.0"
   },
   "scripts": {
-    "test": "grunt travis --verbose"
```
Comment (borek2) [inline package.json]: Why is travis removed? Will it work without?

**Comment by borek2** [comment]
Nice going! How can we see a coverage chart with the tests executed?

File: package.json (hunk 1 above)
Comment (DaanvanderValk) [inline package.json]: Fixed.

File: Gruntfile.js [hunk 2]
```diff
@@ -82,10 +97,11 @@ module.exports = function(grunt) {
...
   grunt.loadNpmTasks('grunt-contrib-watch');
-  grunt.loadNpmTasks('grunt-mocha');
+  grunt.loadNpmTasks("grunt-coveralls");
 
   // Default task.=
-  grunt.registerTask('default', ['jshint', 'qunit', 'clean', 'concat', 'uglify', 'mocha']);
-  grunt.registerTask('travis', ['qunit', 'jshint', 'mocha']);
+  grunt.registerTask('default', ['jshint', 'clean', 'concat', 'uglify', 'qunit']);
+  grunt.registerTask('test', ['qunit']);
+  grunt.registerTask('travis', ['qunit', 'jshint']);
```
Comment (agudek) [inline Gruntfile.js]: qunit is defined in all 3 task types. Is this correct?

Comment (lvdoorn) [inline Gruntfile.js]: This is correct. The 'default' task will be run when executing 'grunt' in a shell. The 'travis' task will be executed on the TravisCI server.

**Pull request merged**